crates/sequencer/src/commit.rs

```diff
@@ -0,0 +1,29 @@
+use blockifier::state::{
+    cached_state::CachedState,
+    state_api::{State as BlockifierState, StateReader as BlockifierStateReader},
+};
+
+pub trait Committer<S>
+where
+    for<'a> &'a mut S: BlockifierState + BlockifierStateReader,
+{
+    fn commit(cached_state: &mut CachedState<&mut S>) {
+        let diff = cached_state.to_state_diff();
+        for (address, class_hash) in diff.address_to_class_hash {
+            let _ = cached_state.state.set_class_hash_at(address, class_hash);
```

Comment:
why do we need `let _` ? to ignore an error? 
why aren't we handling the error if any?